Saml/Saml.cs

```diff
@@ -172,6 +195,12 @@ public string GetNameID()
 			return node.InnerText;
 		}
 
```

Comment:
Is this a Microsoft specific attribute, not sure if it should be here, or somewhere else. I'll come over and discuss this one with you.